projects/aca-playwright-shared/src/page-objects/components/viewer.component.ts

```diff
@@ -52,6 +53,11 @@ export class ViewerComponent extends BaseComponent {
     await this.viewerLocator.waitFor({ state: 'visible', timeout: timeouts.medium });
   }
 
+  async waitForViewerContentToLoad(): Promise<void> {
```

Comment:
We have same method just above as waitForViewerToOpen, if need you can add spinner wait into the same.